docs/essentials/highlight.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Highlight'
+---
+
+Storybook's [Highlight](https://storybook.js.org/addons/@storybook/addon-highlight/) addon allows you to highlight specific DOM nodes within your story. You can use it to call attention to particular parts of the story.
+
+![](highlight.png)
+
+This addon can be used to enhance other addons. For example, [Accessibility](https://storybook.js.org/addons/@storybook/addon-a11y/) addon uses it to highlight DOM nodes that are failing accessibility checks.
+
+## Apply or clear highlights
+
+Highlight DOM nodes by emitting the `HIGHLIGHT` event from within a story or an addon. The event payload must contain a list of selectors you want to highlight.
+
+```js
+import React, { useEffect } from 'react';
+import { useChannel } from '@storybook/addons';
+import { HIGHLIGHT, RESET_HIGHLIGHT } from '@storybook/addon-highlight';
+import { MyComponent } form './MyComponent';
+
+export default { component: MyComponent };
+
+export const MyStory = () => {
+  const emit = useChannel({});
+
+  useEffect(() => {
+    emit(HIGHLIGHT, {
+      elements: ['.title', '.subtitle'],
+    });
+  }, []);
+
+  return <MyComponent />;
+};
+```
```

Comment:
@winkerVSbecks if you wouldn't mind grabbing the code here and creating a snippet file in the react folder, probably calling it `component-story-highlight-addon.js.mdx` and make some adjustments so that it aligns with other existing snippets, for example:
```js
// MyComponent.stories.js|jsx

import React, { useEffect } from 'react';
import { useChannel } from '@storybook/addons';
import { HIGHLIGHT, RESET_HIGHLIGHT } from '@storybook/addon-highlight';
import { MyComponent } form './MyComponent';

export default { 
/* 👇 The title prop is optional.
  * See https://storybook.js.org/docs/react/configure/overview#configure-story-loading
  * to learn how to generate automatic titles
  */
  title: 'MyComponent',
  component: MyComponent
};

export const MyStory = () => {
  const emit = useChannel({});
  useEffect(() => {
    emit(HIGHLIGHT, {
      elements: ['.title', '.subtitle'],
    });
  }, []);
  return <MyComponent />;
};
```
The reasoning behind this is that it provides a bit of clarity in terms of framework usage.

Reply:
@jonniebigodes TBH this is a framework agnostic addon. Can I tweak the code sample to clarify that? If not, I'll move it over.

Reply:
Marking this as solved as @winkerVSbecks moved the snippets to the proper place and provided additional framework examples.